packages/orbit-components/src/SkipNavigation/index.js

```diff
@@ -23,12 +23,14 @@ const StyledNavigation = styled.div`
   ${({ show }) =>
     !show &&
     css`
-      clip: rect(1px, 1px, 1px, 1px);
+      border: 0;
+      clip: rect(0 0 0 0);
+      height: 1px;
+      margin: -1px;
       overflow: hidden;
+      padding: 0;
       position: absolute;
-      pointer-events: none;
-      opacity: 0;
-      width: 0;
+      width: 1px;
```

Comment:
Considering that you're changing this, could you also add `white-space: nowrap` just in case? Because it's included in `sr-only` utilities.

Reply:
i'll add it, but i used [this information]( https://accessibility.18f.gov/hidden-content/), so it's seems correct

Reply:
My bad, your source looks good 👍 Either way the `white-space` rule won't hurt, so it's fine to leave it.